lib/webhdfs/client_v1.rb

```diff
@@ -37,7 +39,7 @@ def ssl_verify_mode=(mode)
       @ssl_verify_mode = mode
     end
 
-    def initialize(host='localhost', port=50070, username=nil, doas=nil, proxy_address=nil, proxy_port=nil, http_headers={})
+    def initialize(host='localhost', port=50070, username=nil, doas=nil, proxy_address=nil, proxy_port=nil, http_headers={}, renew_token_hr=nil)
```

Comment:
`_hr` seems to mean `hour`, but it's confusing. Could you use `hour` instead of the abbreviation?